modules/duo_blocks_cta/duo_blocks_cta.module

```diff
@@ -23,11 +23,14 @@ function duo_blocks_cta_preprocess_block(&$variables, $hook) {
     // Though we don't define any styles for the .button class in this module,
     // let's go ahead and add that class, assuming that the front-end theme can
     // use it.
+
+  if (!empty($variables['content']['field_cta_links']['#items'])) {
```

Comment:
@jthompsonDuo , lines 27-29 need to be indented by two spaces each.

Reply:
updated. please take another look. thx.